File: app/src/main/java/com/nut2014/kotlintest/view/TestActivity.java
package com.nut2014.kotlintest.view;

import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ImageView;
import android.widget.Spinner;
import androidx.appcompat.app.AppCompatActivity;
import com.nut2014.kotlintest.R;
import org.jetbrains.annotations.NotNull;

import java.util.ArrayList;
import java.util.List;

public class TestActivity extends AppCompatActivity implements CoverFragment.OnFragmentInteractionListener {




    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);



        showSelect();

    }

    private void showSelect() {


    }

    @Override
    public void onFragmentInteraction(@NotNull Uri uri) {

    }

    @Override
    public void onPointerCaptureChanged(boolean hasCapture) {

    }
}

File: app/src/main/java/com/nut2014/kotlintest/view/TestActivity.kt
package com.nut2014.kotlintest.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.nut2014.kotlintest.R
import com.nut2014.kotlintest.utils.UserDataUtils

class TestActivity : AppCompatActivity() {
    private val type = 0


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)

        showSelect()

    }

    private fun showSelect() {
        val abc = if (type > 0) UserDataUtils.getId() else 0

    }


}
